Dockerfile

```diff
@@ -36,7 +34,9 @@ RUN gem install bundler -v 2.5.6 --no-doc
 # Install the latest and active gem dependencies and re-run
 # the appropriate commands to handle installs.
 COPY --chown=postal Gemfile Gemfile.lock ./
-RUN bundle install
+RUN bundle install \
+    && rm -rf /usr/local/bundle/cache/*.gem \
+    && rm -rf /opt/postal/.bundle/cache
```

Comment:
I'm not sure how much this would upset doing development work, perhaps if version or branch is detected then do the rm so it only affects shipped images?